src/reason_pprint_ast.ml

```diff
@@ -3014,6 +3011,23 @@ class printer  ()= object(self:'self)
         in
         let rightItm = self#simplifyUnparseExpr rightExpr in
         Some (label ~space:forceSpace (atom prefixStr) rightItm)
+      | (Infix s, [(_, e1); (_, e2)]) when s.[0] = '#' ->
```

Comment:
> Are you saying the naming / AST node is wrong? I just tried and add x##y z##q parses and prints correctly.

Correct, I'm saying we incorrectly classify it as `Infix`. If we leave it as is, there should at least be a comment saying "This isn't *actually* infix, it's simple". You might be able to easily change it to `Simple` though instead of `Infix` in this diff.